fish_speech/webui/manage.py

```diff
@@ -131,6 +132,34 @@ def change_label(if_label):
         yield build_html_ok_message("Nothing")
 
 
+def change_decoder_config(decoder_model_path):
+    if "vits" in decoder_model_path:
+        choices = ["vits_decoder_finetune", "vits_decoder_pretrain"]
+        return {"__type__": "update", "choices": choices, "value": choices[0]}
```

Comment:
Returning `gr.Dropdown(choices=choices, value=choices[0])` should be a better solution,

Reply:
okay, fix it.